bigtable/client/client_options_test.cc

```diff
@@ -46,3 +46,89 @@ TEST(ClientOptionsTest, EditCredentials) {
   EXPECT_EQ(typeid(grpc::InsecureChannelCredentials()),
             typeid(client_options_object.credentials()));
 }
+
+TEST(ClientOptionsTest, SetGrpclbFallbackTimeout) {
+  bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
+  client_options_object.SetGrpclbFallbackTimeout(5);
+  grpc::ChannelArguments c_args = client_options_object.channel_arguments();
+  grpc_channel_args test_args = c_args.c_channel_args();
+  EXPECT_EQ(static_cast<size_t>(2), test_args.num_args);
```

Comment:
I think I suggested using:

```
EXPECT_EQ(2UL, test_args.num_args);
```

here?  Why do you prefer the cast?

Reply:
Misunderstanding. I thought you want we to use static_cast and I also read on stack overflow that for class objects static_cast is preferred.